src/pybind/mgr/nfs/module.py

```diff
@@ -130,3 +130,37 @@ def _cmd_nfs_cluster_config_set(self, cluster_id: str, inbuf: str) -> Tuple[int,
     def _cmd_nfs_cluster_config_reset(self, cluster_id: str) -> Tuple[int, str, str]:
         """Reset NFS-Ganesha Config to default"""
         return self.nfs.reset_nfs_cluster_config(cluster_id=cluster_id)
+
+    def fetch_nfs_export_obj(self):
+        return self.export_mgr
+
+    def fetch_cluster_obj(self):
+        return self.nfs
+
+    def is_active(self) -> bool:
+        return True
+
+    def export_ls(self) -> List[Dict[Any, Any]]:
+        return self.export_mgr.list_all_exports()
+
+    def export_get(self, cluster_id: str, export_id: int) -> Dict[Any, Any]:
+        return self.export_mgr.get_export_by_id(cluster_id, export_id)
+
+    def export_rm(self, cluster_id: str, pseudo: str) -> None:
+        self.export_mgr.delete_export(cluster_id=cluster_id, pseudo_path=pseudo)
+
+    def daemon_ls(self) -> List[Dict[Any, Any]]:
+        return self.nfs.list_daemons()
+
+    def cluster_ls(self) -> List[str]:
+        return [
+            {
+                'pool': NFS_POOL_NAME,
+                'namespace': cluster_id,
+                'type': 'orchestrator',
+                'daemon_conf': None,
+            } for cluster_id in available_clusters()
+        ]
+
+    def cluster_fsals(self) -> List[str]:
+        return ['CEPH', 'RGW']
```

Comment:
I wonder if we need a remote call for static data 🤔 What about simply exporting/importing a constant `NFSGANESHA_SUPPORTED_FSALS = ['CEPH', 'RGW']`?

Reply:
Yes, importing is better. I have added the patch for it but importing from `nfs` module throws [attribute error](https://gist.github.com/varshar16/61ac26426bbe5f5f562ebb14bcd0f548). I have not looked into fixing it. For now I have defined constants in both nfs module and dashboard.